src/components/UI/Typography/Typography.js

```diff
@@ -87,3 +98,11 @@ export const TextBookmark = styled.p`
 	font-family: 'Courier New', Courier, monospace;
 	font-weight: bolder;
 `;
+
+export const TextTitle = styled.h4`
+	justify-content: space-between;
+	font-size: 1.6rem;
+	color: white;
+	font-family: 'Courier New', Courier, monospace;
```

Comment:
Not sure if you use this font-family everywhere but if you do, it might be better to set it on a parent element so that you don't have to set in on each single element.

Reply:
It's already in my Global Styles, don't know why I added it here too:-)